lib/core/test/integration/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1015,6 +1018,34 @@ spec = do
         r <- request @ApiFee ctx (postTxFeeEp w) Default payload
         expectResponseCode @IO HTTP.status404 r
         expectErrorMessage (errMsg404NoWallet $ w ^. walletId) r
+
+    it "TRANS_CREATE_10, TRANS_ESTIMATE_10 - \
+        \Cannot post/estimate tx when max tx size reached (single output)" $ \ctx -> do
+        (wSrc, _, payload) <- fixtureMaxTxSize ctx (11, 1_000_000) 10_000_001
+        fee <- request @ApiFee ctx (postTxFeeEp wSrc) Default payload
+        tx <- request @(ApiTransaction t) ctx (postTxEp wSrc) Default payload
+        verify fee
+         [ expectResponseCode HTTP.status403
+         , expectErrorMessage (errMsg403TxTooBig 10)
+         ]
+        verify tx
+         [ expectResponseCode HTTP.status403
+         , expectErrorMessage (errMsg403TxTooBig 10)
+         ]
+
+    it "TRANS_CREATE_10, TRANS_ESTIMATE_10 - \
+        \Cannot post/estimate tx when max tx size reached (multi output)" $ \ctx -> do
+        (wSrc, _, payload) <- fixtureNInputs ctx (11, 1_000_000) 1
```

Comment:
the body here is the same as in above, the only difference is fixtureMaxTxSize in the above and description. Maybe it is good idea to construct one thing and invoke it with fixture and description rather than duplicate the same stuff?

Reply:
Made it as one `describe` test. I've also moved those tests as specific to `jormungandr` and `http-bridge` since they now behave differently after recent changes related to `estimateMaxNumberOfInputs` (I think that's the cause).